scripts/main.js

```diff
@@ -71,4 +71,90 @@ const departments = {
     }
 }
 
-console.log(departments);
\ No newline at end of file
+// console.log(departments);
+//
+// // Opdracht 1a.
+// console.log("De afdeling Sales heeft " + departments.sales.numberOfEmployees + " medewerkers");
+//
+// // Opdracht 1b.
+// console.log("Marketing is een leuke afdeling om te werken. " + departments.marketing.description);
+//
+// // Opdracht 1c.
+// console.log("De afdeling Customer Service heeft " + departments["customer-service"].numberOfEmployees + " medewerkers");
+//
+// // Opdracht 1d.
+// console.log("Sales is een uitdagende afdeling om te werken als Verkoopmanager. " + departments.sales.jobs[1].description);
+
+
+//Opdracht 2a.
+
+const chosenDepartment = prompt('Over welke afdeling wil je meer informatie? Kies uit: [marketing / sales / customer-service]');
+
+let output = ''
+
+if (chosenDepartment === 'marketing') {
+    output = 'marketing';
+    console.log(['marketing'] + ' is een leuke afdeling om te werken. Er werken op dit moment ' + departments.marketing.numberOfEmployees + ' medewerkers.');
+        // console.log('Je koos Marketing. ' + departments.marketing.description);
+} else if (chosenDepartment === 'sales') {
+    output = 'sales'
+    console.log(['sales'] + ' is een leuke afdeling om te werken. Er werken op dit moment ' + departments.sales.numberOfEmployees + ' medewerkers.');
+    // console.log('Je koos Sales. ' + departments.sales.description);
+} else if (chosenDepartment === 'customer-service') {
+    output = 'customer service'
+    console.log(['customer-service'] + ' is een leuke afdeling om te werken. Er werken op dit moment ' + departments['customer-service'].numberOfEmployees + ' medewerkers.');
+    // console.log('Je koos Customer Service. ' + departments['customer-service'].description);
+} else {
+    console.error('Ongeldige keuze. Probeer het opnieuw door de pagina te verversen');
+}
```

Comment:
Hier zou je een switch voor kunnen gebruiken om het wat makkelijker te maken en vervolgens kun je dan ook een hoop code dubbeling reduceren
```suggestion
switch(chosenDepartment) {
    case "marketing":
    case "sales":
    case "customer-service":
        output = chosenDepartment;
        console.log(`${chosenDepartment} is een leuke afdeling om te werken. Er werken op dit moment ${departments[chosenDepartment].numberOfEmployees} medewerkers.');
        // console.log(`Je koos ${chosenDepartment}. ${departments.marketing.description}`);
        break;
    default:
        console.error('Ongeldige keuze. Probeer het opnieuw door de pagina te verversen');
        break;
}
```